homeassistant/components/switcher_kis/quality_scale.yaml

```diff
@@ -0,0 +1,78 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: The integration uses entity services.
+  appropriate-polling:
+    status: exempt
+    comment: The integration does not poll.
+  brands: done
+  common-modules: done
+  config-flow-test-coverage:
+    status: todo
+    comment: make sure flows end with created entry or abort
+  config-flow: done
```

Comment:
Why do we do `default=""`

Reply:
Fixed at https://github.com/home-assistant/core/pull/143607